docs/topics/settings.rst

```diff
@@ -1116,17 +1116,6 @@ multi-purpose thread pool used by various Scrapy components. Threaded
 DNS Resolver, BlockingFeedStorage, S3FilesStore just to name a few. Increase
 this value if you're experiencing problems with insufficient blocking IO.
 
-.. setting:: REDIRECT_MAX_TIMES
```

Comment:
Removed to avoid errors about duplicate definitions, such as
```
/home/travis/build/scrapy/scrapy/docs/topics/settings.rst:1119:duplicate setting description of REDIRECT_MAX_TIMES, other instance in topics/downloader-middleware
```
https://travis-ci.org/github/scrapy/scrapy/jobs/673570891#L348